sdk/clientcore/core/src/main/java/io/clientcore/core/http/pipeline/HttpInstrumentationPolicy.java

```diff
@@ -148,26 +148,25 @@
 public final class HttpInstrumentationPolicy implements HttpPipelinePolicy {
     private static final ClientLogger LOGGER = new ClientLogger(HttpInstrumentationPolicy.class);
     private static final HttpInstrumentationOptions DEFAULT_OPTIONS = new HttpInstrumentationOptions();
-    private static final String LIBRARY_NAME;
-    private static final String LIBRARY_VERSION;
-    private static final LibraryInstrumentationOptions LIBRARY_OPTIONS;
+    private static final String SDK_NAME;
+    private static final String SDK_VERSION;
+    private static final SdkInstrumentationOptions SDK_OPTIONS;
     private static final TraceContextSetter<HttpHeaders> SETTER
         = (headers, name, value) -> headers.set(HttpHeaderName.fromString(name), value);
 
     static {
         Map<String, String> properties = getProperties("core.properties");
-        LIBRARY_NAME = properties.getOrDefault("name", "unknown");
-        LIBRARY_VERSION = properties.getOrDefault("version", "unknown");
-        LibraryInstrumentationOptions libOptions
-            = new LibraryInstrumentationOptions(LIBRARY_NAME).setLibraryVersion(LIBRARY_VERSION)
-                .setSchemaUrl("https://opentelemetry.io/schemas/1.29.0");
+        SDK_NAME = properties.getOrDefault("name", "unknown");
+        SDK_VERSION = properties.getOrDefault("version", "unknown");
+        SdkInstrumentationOptions sdkOptions = new SdkInstrumentationOptions(SDK_NAME).setLibraryVersion(SDK_VERSION)
```

Comment:
maybe it's too much "options in your options", but now that we have `UserAgentOptions` should we use it in `SdkInstrumentationOptions`?